e2e/test/scenarios/embedding/embedding-smoketests.cy.spec.js

```diff
@@ -162,7 +162,13 @@ describe("scenarios > embedding > smoke tests", { tags: "@OSS" }, () => {
               element.textContent ===
               "You can change the font with a paid plan.",
           );
-          cy.findByText("Download buttons").should("not.exist");
+
+          cy.findByText(
+            object === "dashboard"
+              ? "Results (csv, xlsx, json, png)"
+              : "Download (csv, xlsx, json, png)",
+          ).should("not.exist");
```

Comment:
I could've matched this with `/csv, xlsx, json, png/` to make it shorter but I prefer using the full string, for ease of find-and-replace.